ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -1606,6 +1606,18 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
         var icon = getIcon(scope.file.name || scope.file.id || '') || {};
         scope.icon = iconURL.replace('page_white', attrs.icon || icon.name || 'page_white');
         scope.type = attrs.type || icon.type || 'Attachment';
+        if(scope.type === 'Image') {
+           var urlDispatcher = new XWiki.Document('URLDispatcher', 'FileManagerCode');
+           var docRef = new XWiki.Document(scope.file.id);
+           scope.icon=urlDispatcher.getURL('get', $.param({
+             entityType: 'attachment',
+             wiki: docRef.wiki,
+             space: docRef.space,
+             page: docRef.page,
+             file: scope.file.name,
+             queryString: 'width=512'
```

Comment:
I would add a comment to explain this query string (limit the size of the downloaded file by resizing the image on the server).